### ISSUE TITLE
[Approval needed] Add a page called Additional Settings

### ISSUE THREAD

**Issue by Cyberavater** [MuntashirAkon/AppManager#369]
Apps from Onboard can be added to the Additional Settings page;
All apps added to the Additional Settings page will be listed here; [1]
There will be a "Settings" icon here (probably at the top left), clicking it, would open the page with various settings that can be applicable to the app; [2]
This page would contain many settings separated with tiles by their types, some examples;

Networking:
◘Internet [This would remove internet access from the app]

Network State:
◘Wifi [Turns on wifi when the app is launched, if not already turned on]

Display:
◘Immersive Mode [#351]

Tracking:
◘Block Trackers

From, [1]-(List of apps) all apps are selectable, selecting which would open their own [2]-(individual settings) that would override the global settings. Both [2] would have a switch at the top left which would make the mentioned settings in that page enabled or disabled.

Of course, this is not even half the suggestion/plan I have with it. I'll add more suggestions, settings, designs (and possibly try to add related documents and methods to do so?) in more detail if the dev thinks it's a good suggestion and he would work on it. Otherwise, I won't waste any time regarding this (even tho adding this short demo took a certain amount of time and effort).

**Comment by MuntashirAkon** [comment]
I've a fixed road map and I'm not sure how well would this feature go with the plan, especially #61 and #321 which would create a lot of opportunities for integrating other features. Therefore, I'll consider this only _after_ the said features are implemented.

**Comment by Cyberavater** [comment]
Fair enough, even I myself found some alternative UIs and ain't sure which would provide the best UX. But options from here should be helpful regardless of that, so I'd add some from time to time (as I already forgot some of them, because of not writing them down).

**Comment by MuntashirAkon** [comment]
Add whatever you find. It'll give us some ideas about what to expect.

**Comment by Cyberavater** [comment]
Sure, I'll update the OP from time to time, if anything is unclear feel free to ask away, even if it's implementation-related. Altho, it's most likely I won't be able to help, but there is a slim chance I might, if not others might pick up and give an alternative suggestion to the hindrance (I've also added/proposed a method to implement three/multiple state switch).
